package.json

```diff
@@ -8,8 +8,10 @@
   "license": "MIT",
   "private": false,
   "scripts": {
-    "build": "npx tsc-bundle ./tsconfig.json && terser assets/js/dist/main.js -c -m -o assets/js/dist/main.js",
-    "lint": "tslint ./tsconfig.json",
+    "build": "npm run lint && npx tsc-bundle ./tsconfig.json && npm run minify",
```

Comment:
Don't you want to use yarn instead of npm ?

Reply:
Fix !